modules/images/dominant-color/class-dominant-color-image-editor-gd.php

```diff
@@ -9,6 +9,11 @@
  * @since 1.2.0
  */
 
+if ( ! class_exists( 'WP_Image_Editor_GD' ) ) {
+	require_once ABSPATH . WPINC . '/class-wp-image-editor.php';
+	require_once ABSPATH . WPINC . '/class-wp-image-editor-gd.php';
+}
+
```

Comment:
Upon further testing, this change shouldn't be needed.

@mukeshpanchal27 Can you remove this change locally and confirm the tests still pass?

If the tests still pass, you can go ahead and revert [3ec7d40](https://github.com/WordPress/performance/pull/572/commits/e76b654aa4314101eb52393acc64196ce5b28fe6).

Reply:
Thanks @costdev. Reverted [3ec7d40](https://github.com/WordPress/performance/pull/572/commits/e76b654aa4314101eb52393acc64196ce5b28fe6).